static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -266,7 +274,7 @@ function WidgetBuilder({
     pathname: currentDashboardId
       ? `/organizations/${orgId}/dashboard/${currentDashboardId}/`
       : `/organizations/${orgId}/dashboards/new/`,
-    query: {...location.query},
+    query: omit({...location.query, ...globalSelectionHeaderParams}, 'period', 'source'),
```

Comment:
Currently, `globalSelectionHeaderParams` has values such as `start: null, end: null`, they appear in the route. Should I be filtering them out? Is this an incorrect usage of propagating global selection properties?

It also sets the query param even if the user doesn't make a selection